llvm/lib/CodeGen/GlobalISel/IRTranslator.cpp

```diff
@@ -2863,6 +2834,53 @@ bool IRTranslator::translateCall(const User &U, MachineIRBuilder &MIRBuilder) {
   return true;
 }
 
+bool IRTranslator::translateCall(const User &U, MachineIRBuilder &MIRBuilder) {
```

Comment:
Could you place this call back in its original place? It makes the diff comparison hard to read otherwise. (I.e., I don't know what the new `translateTargetIntrinsic` is doing.)

Reply:
done